test/distributed/test_c10d.py

```diff
@@ -1888,6 +1887,24 @@ def forward(self, x):
         return F.softmax(x, dim=1).to(dev0)
 
 
+class ConvNet(nn.Module):
+    def __init__(self, gpus, layouts, dtypes):
+        super(ConvNet, self).__init__()
+        self.gpus = gpus
+        self.dtypes = dtypes
+        self.conv0 = torch.nn.Conv2d(8, 16, (2, 2)).to(gpus[0]).to(memory_format=layouts[0]).to(dtypes[0])
```

Comment:
nit: you can pass all these to a single `.to()` call

Reply:
could have sworn i tried to do that at one point and it failed...oh well, i tried it again and now it works.  Updated.